common-proxy/src/main/java/de/maxhenkel/voicechat/config/ProxyConfig.java

```diff
@@ -16,27 +16,31 @@ public ProxyConfig(ConfigBuilder builder) {
 
         port = builder
                 .integerEntry("port", -1, -1, 65535,
-                        "The port of the voice chat proxy server",
-                        "Setting this to \"-1\" sets the port to the proxies port"
+                        "The port number to use for the voice chat communication.",
+                        "Audio packets are always transmitted via the UDP protocol on the port number",
+                        "specified here, independently of other networking used for the game server.",
+                        "Set this to '-1' to use the same port number that is used by the proxy.",
+                        "However, it is strongly recommended NOT to use the same port number because UDP on",
+                        "it is also used by default for the server query. Doing so may crash the server!"
```

Comment:
The part that it is not recommended doesn't apply for the proxy, as its default port isn't 25565 and the port -1 is used by default.

Reply:
Oh right. I should have figured that out myself, especially since it's clearly written just five lines above.